### ISSUE TITLE
Stopped working likely due to Cloudflare DDoS Protection

### ISSUE THREAD

**Issue by TheRedMelon** [Dadido3/Pixelcanvas.io-Custom-Client#9]
Recently stopped working in the past week, I believe Cloudflare DDoS protection was added which is likely causing problems.  No pixels are successfully sent and there error message field at the bottom shows just "Error:".  Probably unrelated but the exit button does not work and it needs to be closed with the Task Manager.

**Comment by Dadido3** [comment]
You are right, i just tested it, and the client is sending a lot of http post requests. That's because for these requests, pixelcanvas.io seems to return a browser verification page, instead of the expected json data.

That verification page contains some obfuscated js challenge from Cloudflare (Looks like it's about adding float numbers). Unfortunately i can't run that challenge, as i don't have a good way to run js from the language i'm using. In hindsight i should have written the program in go or something else, that would have made a lot of things easier now.

As the program doesn't work anymore, i'm going to put a statement into the `README.md`

Because of the exit bug: That's just a small bug in the event processing, which is triggered by the slowdown of eventhandling by all the http requests. There is no use to fix that now, as the bot isn't working anyways.

Another note without promising anything:
I thought about rewriting that bot in go. Or atleast the networking, and reuse the current bot as user interface. But that is a bit of a bigger project, the effort and use isn't justified by the current popularity of pixelcanvas. Looking over the central regions of the canvas, there's hardly been any change to the pixelarts. And then i also read something about the website being shutdown soon, but i have no idea if that was just a rumor.

tl;dr
Everything was nice while it lasted, but the bot/custom-client is now broken.
